pubtools/pulplib/_impl/model/repository/repo_lock.py

```diff
@@ -0,0 +1,222 @@
+import json
+import random
+import string
+from time import sleep
+import os
+import logging
+
+from datetime import datetime, timedelta
+from more_executors.futures import f_map
+from requests.exceptions import HTTPError
+
+LOCK_CLAIM_STR = "pulplib-lock-claim-"
+
+LOCK_VALID_FROM_OFFSET = int(os.getenv("PULPLIB_LOCK_VALID_FROM_OFFSET", "10"))
+LOCK_EXPIRATION_OFFSET = int(os.getenv("PULPLIB_LOCK_EXPIRATION_OFFSET", "1800"))
+LOCK_EXPIRATION_OFFSET_MIN = int(os.getenv("PULPLIB_LOCK_EXPIRATION_OFFSET_MIN", "300"))
+LOCK_EXPIRATION_OFFSET_MAX = int(
+    os.getenv("PULPLIB_LOCK_EXPIRATION_OFFSET_MAX", "216000")
+)
+
+AWAIT_ACTIVATION_SLEEP = os.getenv("PULPLIB_AWAIT_ACTIVATION_SLEEP", 10)
```

Comment:
This still has the same bug as above used to (it will return an int if the env var is not set, and a str if the env var is set).  I suppose if you set the env var, the later sleep(AWAIT_ACTIVATION_SLEEP) will crash.